config.ini

```diff
@@ -40,8 +40,8 @@ indexes = ["canton", "court", "chamber", "date"]
 [postgres]
 ip = localhost
 port = 5432
-user = readonly
-password = readonly
+user = postgres
+password = postgres
```

Comment:
please don't commit this, but just keep it locally.

Reply:
Yeah sorry this was a mistake, fixed it now.